pkg/lib/editor/controllers.go

```diff
@@ -210,6 +210,16 @@ func validateConfigBundle(opts *ServerOptions) func(http.ResponseWriter, *http.R
 			Certificates: configBundle.Certificates,
 		}
 
+		// Write database.pem needed for db validation, if any
+		if database_pem_bytes, ok := configBundle.Certificates["database.pem"]; ok {
+			err = ioutil.WriteFile("/tmp/database.pem", database_pem_bytes, 0644)
```

Comment:
Simultaneous requests to this endpoint will interfere with each other. If this is needed (to have the pem content into a file) this should be done in `DatabaseFieldGroup.Validate`, using a randomly generated temp file name.

Reply:
I've updated the commit to instead, write the certificate to a unique temp file in `/tmp`, and set sslrootcert to that.
So simultaneous requests _should_ generate different filenames.